dace/frontend/python/astutils.py

```diff
@@ -663,3 +663,11 @@ def create_constant(value: Any, node: Optional[ast.AST] = None) -> ast.AST:
         newnode = ast.copy_location(newnode, node)
 
     return newnode
+
+
+def escape_string(value: str):
+    """ Converts special Python characters in strings back to their parsable version (e.g., newline to ``\n``) """
+    if sys.version_info >= (3, 0):
+        return value.encode("unicode_escape").decode("utf-8")
+    # Python 2.x
+    return value.encode('string_escape')
```

Comment:
Isn't this a bit pointless? I think that DaCe is completely incompatible with Python 2.

Reply:
True